tests/frontend/grpc/echo_tensor_worker.py

```diff
@@ -0,0 +1,53 @@
+#  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# Usage: `TEST_END_TO_END=1 python test_tensor.py` to run this worker as tensor based echo worker.
+
+
+import uvloop
+
+from dynamo.llm import ModelInput, ModelRuntimeConfig, ModelType, register_llm
+from dynamo.runtime import DistributedRuntime, dynamo_worker
+
+
+@dynamo_worker(static=False)
+async def echo_tensor_worker(runtime: DistributedRuntime):
+    component = runtime.namespace("test").component("tensor")
+    await component.create_service()
+
+    endpoint = component.endpoint("generate")
+
+    model_config = {
+        "name": "tensor",
+        "inputs": [
+            {"name": "dummy_input", "data_type": "Bytes", "shape": [-1]},
+        ],
+        "outputs": [{"name": "dummy_output", "data_type": "Bytes", "shape": [-1]}],
+    }
+    runtime_config = ModelRuntimeConfig()
+    runtime_config.set_tensor_model_config(model_config)
```

Comment:
Just curious, I notice we register dummy I/O in model config here and on client side send arbitrary input names with differing data types.

In future, do we plan any validation on frontend with the registered model config to validate that the I/O in request matches the I/O in the model config?

Reply:
Frontend side validation is not being planned. Could do if requested